pom.xml

```diff
@@ -105,6 +105,16 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.apache.hadoop</groupId>
+                <artifactId>hadoop-common</artifactId>
+                <version>3.3.1</version>
```

Comment:
we should extract a {HADOOP_VERSION} for them (maybe later)

Reply:
ok